crate/operator/create.py

```diff
@@ -768,6 +771,30 @@ async def create_statefulset(
                 logger,
             ),
         )
+        policy = PolicyV1beta1Api(api_client)
```

Comment:
I think lets add a comment here saying what a PDB is, as it's not super clear what it is for w/o reading the docs.

Something along the lines of "A Pod Distruption Budget ensures that when performing k8s cluster maintenance (i.e. upgrades), we make sure to not disrupt more than X pods in a StatefulSet at a time. This means that when doing a rolling restart, we'll wait for all pods to come back before restarting the next k8s node."